peg/Parser.hx

```diff
@@ -318,6 +335,11 @@ class Parser {
 
 		parseArguments(ctx, fn);
 
+		var rt = parseDocTagReturnType(fn);
+		if (rt != null && fn.returnType == TMixed) {
```

Comment:
makes sense to check `fn.returnType` before `parseDocTagReturnType(fn)` to avoid unnecessary parsing.

Reply:
done in 09f1f4c